babel.config.js

```diff
@@ -3,6 +3,14 @@ module.exports = {
   plugins: [
     '@babel/transform-runtime',
     '@babel/proposal-class-properties',
-    '@babel/proposal-object-rest-spread'
+    '@babel/proposal-object-rest-spread',
+    [
+      'module-resolver',
```

Comment:
I think for storybook, we'll want to set up any aliases in the [webpack.config.js](https://github.com/pricelinelabs/design-system/blob/next/storybook/webpack.config.js) file instead of Babel, so that the  the Babel transpiled dist files aren't affected

Reply:
@jxnblk, with the latest rebase w. `next-deps` branch, the `babel-plugin-module-resolver` code is not needed & it's been reverted. 👍